protobuf/key_attributes.proto

```diff
@@ -18,62 +18,59 @@ syntax = "proto3";
 
 package key_attributes;
 
+message KeyAttributesProto {
```

Comment:
I was thinking about removing the `Proto` part, but just as we use both the protobuf-generated structs and non-protobuf specific ones, others might do the same in client libraries, and having them named identically might be confusing.

Reply:
Also, do you want the key bits here or not? I'm ok with it either way, even if it's in here as well the default value is 0

Reply:
> as we use both the protobuf-generated structs and non-protobuf specific ones, others might do the same in client libraries, and having them named identically might be confusing.

I think the generation problems of those files should not influentiate the way they are named. We could use the full namespace of the generated files in Rust or use `as` for example. My thinking was, as they are our operation contracts, they should be the most "important" parts of our stack and should influentiate the rest but not the other way around.

Reply:
I would prefer having the `key_bits` out of the structure as it seems a be a duplication of information.

Reply:
> as it seems a be a duplication of information.

It's only duplication of information in `ImportKey`, because you have the key data available (and even then, it's not really explicitly duplicated, you can just infer it by parsing the key structure which otherwise might not be necessary), but in other cases that I can think of (create key, key agreement/derivation, getting a description of keys you have available) it's not duplicated.